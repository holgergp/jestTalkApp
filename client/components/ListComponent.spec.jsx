import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import List from "./ListComponent.jsx";

describe("ListComponent", () => {
  it("shallow renders", () => {
    const testArray = ["1", "2"];
    const renderer = new ShallowRenderer();
    const rendered = renderer.render(<List values={testArray} />);
    const shallowRendered = renderer.getRenderOutput();
    expect(shallowRendered).toMatchSnapshot();
  });
});
