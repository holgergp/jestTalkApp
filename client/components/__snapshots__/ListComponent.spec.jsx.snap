// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListComponent shallow renders 1`] = `
<dl>
  <dt>
    <TextComponent
      value="1"
    />
  </dt>
  <dt>
    <TextComponent
      value="2"
    />
  </dt>
</dl>
`;
